atc/db/migration/migrations/1561054662_users_table.up.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+  CREATE TABLE users (
+    "id" serial NOT NULL PRIMARY KEY,
+    "username" text NOT NULL,
+    "connector" text NOT NULL,
+    "last_login" timestamp with time zone DEFAULT now() NOT NULL
+  );
+  ALTER TABLE ONLY users ADD CONSTRAINT user_unique UNIQUE (username,connector);
```

Comment:
When we auth with dex we get back a unique `sub` in the token. Can we add a column to track this value and unique on it, instead of the username/connector combo.

Reply:
Do you mean to add a new column `sub`?  Or completely replace the columns `username` or `connector`?
It seems like the current `username/connector` is more readable and allow us to trace the source the authentication.
WDYT?

Reply:
lets keep the username connector but also include a `sub` or `user_id` and unique on that